boards/arm/nrf9160_innblue22/doc/index.rst

```diff
@@ -0,0 +1,145 @@
+.. _nrf9160_innblue22:
+
+nRF9160 INNBLUE22
+#################
+
+Overview
+********
+
+The "nRF9160_innblue22" is a cellular IoT sensor development board, which
```

Comment:
```suggestion
The nRF9160 innblue22 is a cellular IoT sensor development board, which
```

This looks a lot like the board name you would type into the console (`nrf9160_innblue22`), which is case sensitive. So I suggest removing the underscore and quotes to keep it consistent with the page title and avoid confusion with the command-line board name.